substra/sdk/models.py

```diff
@@ -109,12 +109,12 @@ class _ObjectiveDataset(schemas._PydanticConfig):
 class _Metric(schemas._PydanticConfig):
     """Metric associated to a testtuple or objective"""
     name: Optional[str]
-    hash_: str = pydantic.Field(..., alias="hash")
+    checksum: str
     storage_address: UriPath
 
     @property
     def key(self):
-        return self.hash_
+        return self.checksum
```

Comment:
Is this still true? I was under the impression that keys and checksum were now always distinct.

Reply:
Good catch! 

A `Metric` doesn't have a `key`: only the hosting `Objective` does.

I removed the property.